modules/samples/getting_started/src/org/wso2/carbon/registry/samples/populator/CategorizeArtifacts.java

```diff
@@ -130,8 +130,9 @@ public static void main(String[] args) throws Exception {
                             }
                             i++;
                         } else {
-                            String category = "Sales";
-                            artifact.setAttribute("overview_category", category);
+                            environment = "Sales";
+                            artifact.setAttribute("categorization_environment", environment);
```

Comment:
@splinter This code segment has also been replaced by a later commit and doesn't appear in the latest version of the file.